tests/test-sources/plugins/bufferlines/bufferline.nix

```diff
@@ -124,10 +125,85 @@
     };
   };
 
-  no-packages = {
-    plugins.bufferline = {
-      enable = true;
-      iconsPackage = null;
+  default-packages =
+    { config, ... }:
+    {
+      plugins.bufferline = {
+        enable = true;
+      };
+
+      assertions = [
+        {
+          assertion =
+            config.extraPlugins != [ ]
+            -> lib.any (
+              x:
+              lib.trace "${x.pname or x.name}" x.pname or x.name != null && x.pname or x.name == "bufferline.nvim"
```

Comment:
No need for a null check here. `null == "foo"` is valid nix and will evaluate to `false`.

I think what you want instead is `x.pname or x.name or null == "bufferline.nvim"`, however in final code you should know if you're checking `name` or `pname`, so just check one not both.